tests/test__repr_svg_.py

```diff
@@ -0,0 +1,131 @@
+import xml.etree.ElementTree as ET
+
+from . import unittest
+from shapely.geometry import Point, MultiPoint, LineString, MultiLineString,\
+    Polygon, MultiPolygon
+from shapely.geometry.collection import GeometryCollection
+
+
+class _Repr_Svg_TestCase(unittest.TestCase):
+
+    def assert_repr_svg_(self, geom, expected):
+        """Helper function to check SVG representation for iPython notebook"""
+        # avoid order-related issues in string comparison by running through
+        # Element string parsing
+        self.assertEqual(ET.tostring(ET.fromstring(geom._repr_svg_())),
+                         ET.tostring(ET.fromstring(expected)))
+
+    def test_empty(self):
+        self.assert_repr_svg_(
+            Point(), 
+            '<svg xmlns="http://www.w3.org/2000/svg" '
+            'xmlns:xlink="http://www.w3.org/1999/xlink" />')
+
+    def test_point(self):
+        self.assert_repr_svg_(
+            Point(6, 7),
+            '<svg width="100.0" height="100.0" '
+            'xmlns:xlink="http://www.w3.org/1999/xlink" '
+            'xmlns="http://www.w3.org/2000/svg"'
+            'viewBox="5.0 6.0 2.0 2.0" preserveAspectRatio="xMinYMin meet">'
+            '<g transform="matrix(1,0,0,-1,0,14.0)">'
+            '<circle cx="6.0" cy="7.0" r="0.06" stroke="#555555" '
+            'stroke-width="0.02" fill="#66cc99" opacity="0.6" />'
+            '</g></svg>')
+
+    def test_multipoint(self):
+        self.assert_repr_svg_(
+            MultiPoint([(6, 7), (3, 4)]),
+            '<svg xmlns="http://www.w3.org/2000/svg" '
+            'xmlns:xlink="http://www.w3.org/1999/xlink" '
+            'width="100.0" height="100.0" '
+            'viewBox="2.88 3.88 3.24 3.24" preserveAspectRatio="xMinYMin meet">'
+            '<g transform="matrix(1,0,0,-1,0,11.0)"><g>'
+            '<circle cx="6.0" cy="7.0" r="0.09720000000000001" '
```

Comment:
When I see something like that `r` value, I worry that it will be platform dependent. On a different system it might be `"0.09720000000000002"`.

The most important thing might be to check that we have two circle elements and that their radii are roughly correct?